src/frontends/tensorflow/src/op/parallel_dynamic_stitch.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_table.hpp"
+#include "openvino/opsets/opset9.hpp"
+
+using namespace std;
+using namespace ov::opset9;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_parallel_dynamic_stitch_op(const NodeContext& node) {
+    // format for inputs: [indices1, indices2, ..., indicesN, data1, data2, ..., dataN]
+    // so we expect at least 2 input and the total number of inputs must be divisible by 2
+    default_op_checks(node, 2, {"ParallelDynamicStitch", "DynamicStitch"});
+    auto in_size = node.get_input_size();
+    TENSORFLOW_OP_VALIDATION(node,
+                             in_size % 2 == 0,
+                             "The total number of inputs to DynamicStitch operation "
```

Comment:
`... to DynamicStitch or ParallelDynamicStitch`

Reply:
fixed